queries/css/locals.scm

```diff
@@ -0,0 +1 @@
+; Placeholder file to get incremental selection to work.
```

Comment:
Wouldn't it be better to fix incremental selection to always work?

Reply:
yeah +1 on that.